src/business_logic/transaction/error.rs

```diff
@@ -121,8 +120,8 @@ pub enum TransactionError {
     CairoRunner(#[from] CairoRunError),
     #[error(transparent)]
     Runner(#[from] RunnerError),
-    #[error(transparent)]
-    StarknetRunner(#[from] StarknetRunnerError),
+    #[error("Invalid Return Data")]
```

Comment:
Maybe we can improve a bit this error message? maybe including the number of values that it is trying to fetch?